src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetectionExecutor.swift

```diff
@@ -104,9 +104,18 @@ final class ExposureDetectionExecutor: ExposureDetectionDelegate {
 			}
 			fatalError("invalid state")
 		}
+
+		var fileURLs: [URL]
+		if writtenPackages.urls.count > 14 {  // This should never happen. Code is just here to be sure
+			logError(message: "Trying to feed too many files into the framework")
+			fileURLs = Array(writtenPackages.urls.prefix(14))
```

Comment:
shouldnt we still call the result handler with an error here or is it safe to proceed with the first 14 items?

Reply:
The condition should never be true. It's just here to be absolutely sure that we don't feed too many files per day. So it's either truncate or fail completely.